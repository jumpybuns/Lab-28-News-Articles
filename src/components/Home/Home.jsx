import React from 'react';
import NewsSearch from '../Search/NewsSearch';

export default function App() {
  return (
    <>
      <p>Now What...</p>
      <NewsSearch />
    </>
  );
}
