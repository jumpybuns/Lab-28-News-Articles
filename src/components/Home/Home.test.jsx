global.fetch = require('node-fetch');
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Home from './Home';

describe('Home component', () => {
  afterEach(() => cleanup());
  it('renders Home', () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});
