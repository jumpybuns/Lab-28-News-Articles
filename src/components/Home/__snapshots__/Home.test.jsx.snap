// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders Home 1`] = `
<DocumentFragment>
  <h1>
    Now What...
  </h1>
  <div>
    <div>
      <input
        type="text"
        value=""
      />
    </div>
    <ul />
  </div>
</DocumentFragment>
`;
